src/components/PriorityPicker/PriorityPicker.js

```diff
@@ -261,10 +261,10 @@ function Priority({
       return value.agent_address === whoami.entry.address
     })
     if (!vote) return
-    archiveVoteOfGoal(vote.address).then(() => {
```

Comment:
This was the source of the "myVote is undefined" bug that would crash the screen, when you archived a goal vote